charts/posthog/templates/worker-deployment.yaml

```diff
@@ -33,31 +33,41 @@ spec:
         {{- end }}
     spec:
       serviceAccountName: {{ template "posthog.serviceAccountName" . }}
+
       {{- if .Values.worker.affinity }}
       affinity:
 {{ toYaml .Values.worker.affinity | indent 8 }}
       {{- end }}
+
```

Comment:
Formatting changes because otherwise this is a giant block of "omg I cannot read it"